package.json

```diff
@@ -4,6 +4,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open",
```

Comment:
[OPTIONAL] 

- [ ] You've used `webpack-dev-server` in your `start scripts` but `webpack-dev-server` is not added as `devDependencies`. You would receive this error; `webpack-dev-server: not found` when you try to run `npm start` to start the application. Include `webpack-dev-server` in your devDependencies to resolve this.